NetflixClone/Classes/Networking/NetworkConfiguration.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  NetworkConfiguration.swift
+//  NetflixClone
+//
+//  Created by Joseph Ching on 2022-09-19.
+//
+
+import Foundation
+
+class NetworkConfiguration {
```

Comment:
```suggestion
enum NetworkConfiguration {
```
It's common to use case-less enum for constants.